src/main/java/uk/gov/ons/ctp/response/action/service/impl/ActionRuleServiceImpl.java

```diff
@@ -19,6 +21,7 @@
 @Slf4j
 public class ActionRuleServiceImpl implements ActionRuleService {
 
+  private static final int TRANSACTION_TIMEOUT = 30;
```

Comment:
Shouldn't this be consistent across the service? Could be read in from config.

Reply:
It can be but is it worth reading from application.yml file?